# Message with all currently detected persons

Header                  header          # Header containing timestamp etc. of this message
DetectedPerson[]        people      # All persons that are currently being detected
